src/App.jsx

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-import { Route, Switch } from 'react-router-dom';
+import React from "react";
+import { Route, Switch } from "react-router-dom";
 
-import { Header } from './commonComponents';
-import BeerPage from './pages/BeerPage/BeerPage';
-import FavoritesPage from './pages/FavoritesPage/FavoritesPage';
-import MainPage from './pages/MainPage/MainPage';
-import { HOME, FAVORITES, BEER_PAGE } from './constants';
-import './sass/App.sass';
+import { Header } from "./commonComponents";
+import BeerPage from "./pages/BeerPage/BeerPage";
+import FavoritesPage from "./pages/FavoritesPage/FavoritesPage";
+import MainPage from "./pages/MainPage/MainPage";
+import { HOME, FAVORITES, BEER_PAGE } from "./constants";
+import "./sass/App.sass";
 
 function App() {
   return (
-    <div className='app'>
+    <div className="app">
       <Header />
       <Switch>
         <Route path={BEER_PAGE}>
```

Comment:
can specify path params {`${BEER_PAGE}/:id`} and get these params by useParams hook